chiselFrontend/src/main/scala/chisel3/core/package.scala

```diff
@@ -2,12 +2,12 @@
 
 package chisel3
 
+/**
+ * These definitions exist to deal with those clients that relied on chisel3.core
```

Comment:
Missing a period at the end of this sentence (I realize it was like this before and you just moved the text :))